sdcm/utils/adaptive_timeouts/load_info_store.py

```diff
@@ -251,6 +253,54 @@ def get(self, operation: str | None, timeout_occurred: bool | None = None):
         return [hit["_source"] for hit in res["hits"]["hits"]]
 
 
+class ArgusAdaptiveTimeoutStore(AdaptiveTimeoutStore):
+    """
+    Report adaptive timeout results to Argus.
+    """
+
+    def __init__(self):
+        self.test_config = TestConfig()
+
+    def send_adaptive_timeout_results_to_argus(self, results: dict):
```

Comment:
How about a dataclass for results, so it in clear what you actually need in the dict?

Reply:
that's the current API, I don't want to mess around with it in this PR

Reply:
Is this method used outside of `store`?

Reply:
no

Reply:
In that case, we can change it and not break the API right?

Reply:
I've added a dataclass, more or less exactly the same, since metrics are still define in other class (which isn't dataclass)